app/va/va_onsite/va_onsite_client.py

```diff
@@ -32,12 +32,13 @@ def post_onsite_notification(self, data: dict):
             response = requests.post(url=f'{ self.url_base }/v0/onsite_notifications',
                                      data=json.dumps(data),
                                      headers=self._build_header())
+
+            self.logger.info(f'onsite_notifications POST response: status_code={response.status_code}, ' +
+                             f'json={response.json()}')
```

Comment:
You don't need the plus sign.
```
>>> print("print this "
... "and this")
print this and this
```

The f-string syntax makes the code harder to follow.  I would use the C formatting syntax, which Python logging statements support:
```
self.logger.info("onsite_notifications POST response: status code=%d, json=%s", response.status, response.json())
```
I think you can use [pprint](https://docs.python.org/3.8/library/pprint.html?highlight=pprint#pprint.pprint) to format the json output.

Reply:
I removed the `+` but I generally prefer to use f-strings and have the variables inline. Is the format you described a better practice, or just personal preference?

Reply:
In this instance, I think it makes the line easier to read.  Either approach works.  I only use the C formatting for logging.

Reply:
I see what you're saying, but I don't think I'll make this change. I prefer having variables inline, it's less confusing to me.